packages/commonwealth/client/scripts/views/pages/view_thread/ViewThreadPage.tsx

```diff
@@ -148,6 +145,12 @@ const ViewThreadPage = ({ identifier }: ViewThreadPageProps) => {
     apiEnabled: !!user?.activeAccount?.address,
   });
 
+  const { data: viewCount = 0 } = useGetViewCountByObjectIdQuery({
```

Comment:
you are returning 0 by default in the getViewCountByObjectId

Reply:
I mean that it is not necessary to doing `viewCount = 0` here because you do it in `getViewCountByObjectId`

Reply:
actually, the default value for viewCount would still be `undefined`, it only gets `0` if `getViewCountByObjectId` gets called and `getViewCountByObjectId` returns `0`.

I added `viewCount = 0` to ensure its always 0 when real value isn't fetched